config/jobs/kubernetes/sig-cloud-provider/aws/ec2-e2e.yaml

```diff
@@ -242,6 +242,75 @@ presubmits:
             requests:
               cpu: 8
               memory: 10Gi
+  - name: pull-kubernetes-e2e-ec2-device-plugin-gpu
```

Comment:
nit: can we please leave `pull-kubernetes-*` for k/k? usually it's `pull-$repo`

I thought for a moment we were adding a new always_run k/k presubmit

Reply:
Sorry my bad. fixing.